CHANGELOG.md

```diff
@@ -15,6 +15,11 @@
 * support for uploading tar to container [#239](https://github.com/softprops/shiplift/pull/239)
 * fix registry authentication to use URL-safe base64 encoding [#245](https://github.com/softprops/shiplift/pull/245)
 * add StopSignal and StopTimeout to ContainerOptionsBuilder [#248](https://github.com/softprops/shiplift/pull/248)
+* make `config` field of `ImageDetails` optional [#264](https://github.com/softprops/shiplift/pull/264)
+* add `container`, `container_config`, `os_version`, `graph_driver`, `root_fs`, `metadata` fields to `ImageDetails` [#264](https://github.com/softprops/shiplift/pull/264)
+* rename `shiplift::rep::Config` to `shiplift::rep::ContainerConfig` [#264](https://github.com/softprops/shiplift/pull/264)
+* add missing fields ([API version 1.41](https://docs.docker.com/engine/api/v1.41/#operation/ImageInspect)) to `ContainerConfig` and make some fields optional [#264](https://github.com/softprops/shiplift/pull/264)
```

Comment:
So from what I've looked none of the fields that were already available were made optional. So this can just say that missing fields were added,

Reply:
Yes, you are right. Fixed.

Thank you for reviewing!